internal/driver/driver.go

```diff
@@ -48,20 +48,22 @@ const (
 )
 
 const (
+	// Key used in the secret store for RTSP credentials.
```

Comment:
start comment with the name, like so
```suggestion
	// rtspAuthKey defines the secretName used for storing RTSP credentials in the secret store.
```